target/linux/mediatek/dts/mt7981b-tenbay-wr3000k.dts

```diff
@@ -0,0 +1,235 @@
+/dts-v1/;
+
+#include "mt7981.dtsi"
+
+/ {
+	model = "Tenbay WR3000K";
+	compatible = "tenbay,wr3000k", "mediatek,mt7981";
+
+	aliases {
+		ethernet0 = &gmac0;
```

Comment:
should drop this line `ethernet0 = &gmac0`
see https://github.com/openwrt/openwrt/pull/17201